tests/ffc/test_us_equity_pricing_loader.py

```diff
@@ -12,287 +12,524 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 """
 Tests for zipline.data.ffc.loaders.us_equity_pricing
 """
-from collections import OrderedDict
-import os
-import sqlite3
 from unittest import TestCase
 
-import bcolz
-import numpy as np
+from bcolz import ctable
+from nose_parameterized import parameterized
+from numpy import (
+    arange,
+    datetime64,
+    float64,
+    full,
+    iinfo,
+    uint32,
+)
+from numpy.testing import assert_array_equal
 import pandas as pd
+from pandas import (
+    DataFrame,
+    DatetimeIndex,
+    Timedelta,
+    Timestamp,
+)
+from pandas.util.testing import assert_index_equal
 from testfixtures import TempDirectory
 
 from zipline.data.adjustment import Float64Multiply
 from zipline.data.equities import USEquityPricing
 from zipline.data.ffc.loaders.us_equity_pricing import (
-    BcolzRawPriceLoader,
-    SQLiteAdjustmentLoader,
+    BcolzDailyBarReader,
+    BcolzDailyBarWriter,
+    SQLiteAdjustmentReader,
+    SQLiteAdjustmentWriter,
 )
+from zipline.finance.trading import TradingEnvironment
+
+UINT_32_MAX = iinfo(uint32).max
+
+# Test calendar ranges over the month of June 2015
```

Comment:
@ehebert I expanded out the test range to the whole month of June because some of the tests I wanted to run needed more than 4 days to test all the boundary conditions.